nwbwidgets/ophys/segmentation.py

```diff
@@ -356,16 +236,6 @@ def route_plane_segmentation(plane_seg: PlaneSegmentation, neurodata_vis_spec: d
         return PlaneSegmentation2DWidget(plane_seg)
 
 
-def show_grayscale_volume(vol: GrayscaleVolume, neurodata_vis_spec: dict):
-    import ipyvolume.pylab as p3
-
-    fig = p3.figure()
-    p3.volshow(vol.data, tf=linear_transfer_function([0, 0, 0], max_opacity=0.1))
-    return fig
```

Comment:
This function wasn't doing anything different from the usual volume shower above, so remapped the extension neurodatatype in the `viz_spec` to point to the other one instead